ui/src/app/pages/login/account-creation/account-creation.spec.tsx

```diff
@@ -102,81 +102,6 @@ it('should handle username validity if name is valid', () => {
   expect(wrapper.exists('#usernameError')).toBeFalsy();
 });
 
-it('should handle invalid Email', () => {
-  const wrapper = component();
-  expect(wrapper.exists('#contactEmail')).toBeTruthy();
-  expect(wrapper.exists('#invalidEmailError')).toBeFalsy();
-  wrapper.find('input#contactEmail').simulate('change',
-      {target: {value: 'username@'}});
-  expect(wrapper.exists('#invalidEmailError')).toBeFalsy();
-});
-
-// TODO remove after we switch to verified institutional affiliation
-it('should display Institution name and role option by default', () => {
-  serverConfigStore.next({...defaultConfig, requireInstitutionalVerification: false});
-  const wrapper = component();
-  const institutionName = wrapper.find('[data-test-id="institutionname"]');
-  expect(institutionName).toBeTruthy();
-  const institutionRole = wrapper.find('[data-test-id="institutionRole"]');
-  expect(institutionRole).toBeTruthy();
-  expect(institutionRole.find('li').length).toBe(AccountCreationOptions.roles.length);
-  expect(institutionRole.find('li').get(0).props.children).toBe(AccountCreationOptions.roles[0].label);
-});
-
-// TODO remove after we switch to verified institutional affiliation
-it('should display Affiliation information if No is selected', () => {
```

Comment:
Academic vs Non-Academic Affiliation is not a concept which exists in new-style affiliation